src/demo/rewireEnterpriseDataApiService.js

```diff
@@ -1,7 +1,37 @@
 import EnterpriseDataApiService from '../../src/data/services/EnterpriseDataApiService';
-import { enrollments, overview } from './data';
+import { filterCompletedLearnerCourses, filteredEnrollments, filterEnrolledLearners, filterUnenrolledRegisteredLearners, getEnrollmentsCsv, overview } from './data';
```

Comment:
nit: This could be cleaned up a bit by having each import on its own line:

```javascript
import {
  filterCompletedLearnerCourses,
  filteredEnrollments,
  filteredEnrolledLearners,
  ...
} from './data';
```